CenterForOpenScience/ember-osf-web
# [ENG-594] Add integrations section

## Purpose

Add the integrations-section to the new logged-out homepage.

## Summary of Changes

- Add `integrations-section` component, versions A and B
- Add logo images to `public/assets`
- Add to acceptance tests

## Screenshots

Version A
<img width="1440" alt="Screen Shot 2019-06-19 at 8 36 03 PM" src="https://user-images.githubusercontent.com/19379783/59810157-fc496780-92d1-11e9-9145-322da7711129.png">

<img width="725" alt="Screen Shot 2019-06-19 at 8 36 26 PM" src="https://user-images.githubusercontent.com/19379783/59810159-feabc180-92d1-11e9-95ef-ce98c1d6983c.png">

<img width="576" alt="Screen Shot 2019-06-19 at 8 36 39 PM" src="https://user-images.githubusercontent.com/19379783/59810162-010e1b80-92d2-11e9-8627-cfe3ead50f76.png">


Version B
<img width="1056" alt="Screen Shot 2019-06-19 at 8 33 45 PM" src="https://user-images.githubusercontent.com/19379783/59810109-bdb3ad00-92d1-11e9-90f6-a79a6b86918c.png">

<img width="653" alt="Screen Shot 2019-06-19 at 8 34 24 PM" src="https://user-images.githubusercontent.com/19379783/59810117-c1dfca80-92d1-11e9-9aaf-9105ff98e1e1.png">

<img width="499" alt="Screen Shot 2019-06-19 at 8 34 12 PM" src="https://user-images.githubusercontent.com/19379783/59810119-c4422480-92d1-11e9-95ab-3301c3343cd8.png">


## Side Effects

`N/A`

## Feature Flags

This component will have a version A and B, so you'll need to switch the `ab_testing_home_page_version_b` feature flag to see either version.

## QA Notes

There is little functionality to this.  The important thing is that it breaks where it should on ipad/ipad mini/mobile browsers.  There are two versions to this component, A and B.  The only real noticeable difference between A and B is the `Get Started` button at the bottom of the component.  To switch between A and B, you'll need to toggle the `ab_testing_home_page_version_b` feature flag in the developer footer.

## Ticket


https://openscience.atlassian.net/browse/ENG-594

# Reviewer Checklist
